src/routes/email/exception-filters/email-does-not-exist.exception-filter.ts

```diff
@@ -5,13 +5,13 @@ import {
   HttpStatus,
 } from '@nestjs/common';
 import { Response } from 'express';
-import { EmailAddressDoesNotExistError } from '@/datasources/email/errors/email-address-does-not-exist.error';
+import { AccountDoesNotExistError } from '@/datasources/account/errors/account-does-not-exist.error';
 
-@Catch(EmailAddressDoesNotExistError)
+@Catch(AccountDoesNotExistError)
 export class EmailAddressDoesNotExistExceptionFilter
```

Comment:
I think this class also needs to be renamed, right? (and its filename)

Reply:
[fa63a4e](https://github.com/safe-global/safe-client-gateway/pull/1050/commits/fa63a4e9950036c1b044c936774359b805c7c62f)